src/cronSelection.js

```diff
@@ -161,16 +171,16 @@ angular.module('angular-cron-jobs').directive('cronSelection', ['cronService', f
             return null;
         }
     };
-}).filter('cronDayName', function() {
+}).filter("cronDayName", function() {
```

Comment:
We can add an optional parameter here to alter it for quartz, example:

`{{value | cronDayName:quartz}}`

```
return function(input, quartz) { //can access quartz setting and make changes accordingly}
```